src/components/character/Donate.module.css

```diff
@@ -1,7 +1,7 @@
 .root {
   position: fixed;
   top: 130px;
-  right: var(--margin);
+  right: calc(var(--margin) - 1cm);
```

Comment:
centimeters have not been used yet in the codebase, please use some of the constants in the variables.css